Do not import commands library as it is not used

It also makes the script not work on python3, but since the script already uses
subprocess in practice, removing commands import is effectively no change and
fixes the python3 compatibility.

Upstream PR: https://github.com/MariaDB/server/pull/1080

From 9c89fd49a757a87ba5899b3548b0fb1d172c0ec3 Mon Sep 17 00:00:00 2001
From: Honza Horak <hhorak@redhat.com>
Date: Wed, 9 Jan 2019 15:05:02 +0100
Subject: [PATCH] Do not import commands library as it is not used

---
 storage/rocksdb/myrocks_hotbackup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/storage/rocksdb/myrocks_hotbackup.py b/storage/rocksdb/myrocks_hotbackup.py
index 69c75b7cbfd..906ba814776 100755
--- a/storage/rocksdb/myrocks_hotbackup.py
+++ b/storage/rocksdb/myrocks_hotbackup.py
@@ -8,7 +8,6 @@ import os
 import stat
 import sys
 import re
-import commands
 import subprocess
 import logging
 import logging.handlers
-- 
2.17.2

